akka-docs/rst/scala/stream/stages-overview.rst

```diff
@@ -587,6 +587,17 @@ when the predicate first return false and then complete.
 
 **completes** when predicate returned false or upstream completes
 
+takeUntil
```

Comment:
You're right. This is confusing... Do you think it's worthwhile to parameterize takeWhile (that will break binary compatibility, right)? I still thing have the option to be inclusive of the predicate is useful, but if I'm just cluttering up the code I can hold off until the next major version?